release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -383,6 +383,22 @@ Starting in {product-title} {product-version}, Operator Lifecycle Manager (OLM)
 Kubernetes 1.22 introduces link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/#customresourcedefinition-v122[several notable changes] to `v1` of the `CustomResorceDefinition` API.
 ====
 
+[id="ocp-4-9-olm-admin-error-reporting"]
+==== Enhanced error reporting for administrators
+Because administrators should not require understanding of the interaction process between the various low-level APIs or access to the OLM pod logs to successfully debug such issues, {product-title} {product-version} introduces the following enhancements in OLM to provide administrators with more comprehensible error reporting and messages:
```

Comment:
s/administrators/cluster administrators/

^ since the removal of the old 1st para was what had previously clarified "cluster administrators".

s/OLM/Operator Lifecycle Manager (OLM)/

^ for first instance.